webpack.config.babel.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
```

Comment:
removed in https://github.com/zero-plus-x/interactive-logo/pull/1/commits/c360687f8769f4753655e6b1e2eaf1ea8e4f26da